front/src/applications/stdcm/views/OSRDCStdcmConfig.tsx

```diff
@@ -56,6 +58,19 @@ export default function OSRDConfig({
 
   const shouldDisplayStdcmResult = selectedTrain !== undefined;
 
+  const enableApplyButton = () => {
```

Comment:
The naming is not correct :) 
```suggestion
  const disabledApplyButton = () => {
```

Reply:
done